examples/systemd/rest-server.service

```diff
@@ -37,6 +34,13 @@ CapabilityBoundingSet=
 LockPersonality=true
 MemoryDenyWriteExecute=true
 NoNewPrivileges=yes
+
+# See the paragraph "All network sockets allocated ..." in the systemd.socket man page
```

Comment:
I'd prefer to just explain why we use `PrivateNetwork` here:
```
As the listen socket is created by systemd via the rest-server.socket unit, it is no longer necessary for rest-server to have access to the host network namespace.
```

Reply:
Done